src/api/libcellml/analysermodel.h

```diff
@@ -56,9 +56,9 @@ class LIBCELLML_EXPORT AnalyserModel
         INVALID, /**< The model is invalid. */
         NLA, /**< The model defines a system of (potentially non-linear) algebraic equations that require an external NLA solver (e.g., KINSOL) to solve. */
         ODE, /**< The model defines a system of ordinary differential equations that require an external ODE solver (e.g., CVODE) to solve. */
-        OVERCONSTRAINED, /**< The model is overconstrained. */
-        UNDERCONSTRAINED, /**< The model is underconstrainted. */
-        UNSUITABLY_CONSTRAINED /**< The model is unsuitably constrained. */
+        OVERCONSTRAINED, /**< The model is overconstrained (i.e. there are too many equations to compute a variable or some variables, irrespective of consistency). */
```

Comment:
suggest:

`The model is determined to be over constrained as there are more equations than unknown variables.`